stdlib/email/message.pyi

```diff
@@ -15,16 +15,20 @@ _PayloadType: TypeAlias = Message | str
 _EncodedPayloadType: TypeAlias = Message | bytes
 _MultipartPayloadType: TypeAlias = list[_PayloadType]
 _CharsetType: TypeAlias = Charset | str | None
-# Type returned by Policy.header_fetch_parse, AnyOf[str | Header]
+# Type returned by Policy.header_fetch_parse, often str or Header.
 _HeaderType: TypeAlias = Any
-_HeaderTypeParam: TypeAlias = str | Header
+# Type accepted by Policy.header_store_parse.
+_HeaderTypeParam: TypeAlias = str | Header | Any
```

Comment:
Since this is used exclusively for parameters, should it be 
```suggestion
_HeaderTypeParam: TypeAlias = str | Header | object
```
Either way maybe also with a comment explaining the use of Any here (even if it's just "read TODO on `Message`")

Reply:
`object` wouldn't be correct, as it would imply that all values are correct, which isn't the case. We just can't express the accepted type properly.